src/test/TestAccount.cpp

```diff
@@ -114,6 +114,18 @@ TestAccount::tx(std::vector<Operation> const& ops, SequenceNumber sn)
     return transactionFromOperations(mApp, getSecretKey(), sn, ops);
 }
 
+TransactionFramePtr
+TestAccount::sorobanTx(std::vector<Operation> const& ops,
+                       std::vector<SecretKey> const& opKeys,
+                       SorobanResources const& resources, uint32_t inclusionFee,
+                       uint32_t resourceFee, std::optional<std::string> memo)
+{
+    loadSequenceNumber();
```

Comment:
I believe with the latest changes to `sorobanTransactionFrameFromOps`, we don't need to do this anymore, since you can now pass expected seqnum directly

Reply:
(though you'd need to rebase/resolve merge conflicts)